scenario.py

```diff
@@ -126,19 +138,19 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         # Number of epochs, mini-batches and fit_batches in ML training
         if "epoch_count" in params:
             self.epoch_count = params["epoch_count"]
-            assert self.epoch_count > 0
+            assert self.epoch_count > 0, "Error: in .yml epoch_count should be > 0"
         else:
             self.epoch_count = 40  # default
 
         if "minibatch_count" in params:
             self.minibatch_count = params["minibatch_count"]
-            assert self.minibatch_count > 0
+            assert self.minibatch_count > 0, "Error: in .yml minibatch_count should be > 0"
```

Comment:
I would replace` Error: in .yml `by `Error in the provided config file, minibatch ...`
This is the same for the other assert messages